src/storage/src/source/postgres.rs

```diff
@@ -437,7 +460,7 @@ async fn postgres_replication_loop_inner(
 struct RowSender {
     sender: Sender<InternalMessage>,
     activator: SyncActivator,
-    buffered_message: Option<(Row, PgLsn, i64)>,
+    buffered_message: Option<(usize, Row, PgLsn, i64)>,
```

Comment:
i also would like this to be a sub-struct but not required

Reply:
you got it!